src/api.js

```diff
@@ -0,0 +1,28 @@
+const fetch = require('node-fetch');
+const defaultValues = require('./defaultValues');
+const debug = require('./debug/debug');
+
+
+const URL = ({ pageId = 1 } ={}) => `https://api.stackexchange.com/2.2/questions?pagesize=100&order=desc&sort=activity&tagged=testcafe&site=stackoverflow&page=${pageId}&access_token=${defaultValues.token}&key=${defaultValues.key}`;
```

Comment:
This URL is very long and complex. Try to construct it from fragments: the root, the API paths, query params. Try to use [url.format](https://nodejs.org/api/url.html#url_url_format_url_options).